bin/lerna.js

```diff
@@ -1,11 +1,6 @@
 #!/usr/bin/env node
 
 var args = process.argv.slice(2);
-if (args.length > 1) {
```

Comment:
√ Done what you literally said. I've also added an "ls" command that I used to test the too-many-arguments warning with :)